build.gradle

```diff
@@ -1,34 +1,66 @@
-buildscript {
-    repositories {
-        gradlePluginPortal()
-        maven {
-            url = "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "$kotlin_version"
+    id "application"
+    id "com.exactpro.th2.gradle.component" version "0.1.1"
+}
+
+group = 'com.exactpro.th2'
+version = release_version
+
+kotlin.jvmToolchain(11)
+
+repositories {
+    maven {
+        name 'Sonatype_snapshots'
+        url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+    }
+
+    maven {
+        name 'Sonatype_snapshots'
+        url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+        metadataSources {
+            mavenPom()
+            artifact()
         }
     }
-    dependencies {
-        classpath "com.exactpro.th2:th2-gradle-plugin:0.0.1-dev-5915772757-13a28ae-SNAPSHOT"
+
+    maven {
+        name 'Sonatype_releases'
+        url 'https://s01.oss.sonatype.org/content/repositories/releases/'
     }
-}
 
-apply plugin: "com.exactpro.th2.common-conventions"
-apply plugin: "com.exactpro.th2.docker-conventions"
+    maven {
+        name 'Sonatype_releases'
+        url 'https://s01.oss.sonatype.org/content/repositories/releases/'
+        metadataSources {
+            mavenPom()
+            artifact()
+        }
+    }
```

Comment:
issue (non-blocking): there are duplicated repositories with different metadata sources. Please, leave only one copy of each repository